flake.nix

```
@@ -202,6 +201,13 @@
 
           };
 
+          semverShell = pkgs.mkShell {
+            inherit CARGO_TARGET_DIR;
+            buildInputs = [
```

I think `fenixStable` needs to be added here, otherwise `cargo-semver-checks semver-checks` fails to find the compiler.

```suggestion
            buildInputs = [
              fenixStable
              (pkgs.cargo-semver-checks.overrideAttrs (final: prev: {doCheck = false;}))
            ] ++ buildDeps;
```